dt-metrics/contacts/sources.php

```diff
@@ -78,7 +78,7 @@ public function scripts() {
                 'milestone_settings' => $milestone_settings,
                 'translations' => [
                     'filter_contacts_to_date_range' => __( "Filter contacts to date range:", 'disciple_tools' ),
-                    'all_time' => __( "All time", 'disciple_tools' ),
+                    'all_time' => _x( "All Time", 'Dates', 'disciple_tools' ),
```

Comment:
Just have them be all without context. All time is a string used in many places, so we are bound to have duplicates

Reply:
This is fixed now too.